### ISSUE TITLE
INLINE_INHERITED_MEMB doesn't respect EXTRACT_PRIVATE=NO

### ISSUE THREAD

**Issue by jwakely** [doxygen/doxygen#6968]
This looks related to #800 but that is apparently fixed, so I'm opening a new issue.

Given this source:
````
/// This is my foo class
struct Foo
{
  /// public typedef
  typedef int value_type;
  /// public function
  void m_impl() { }
};

/// This is my bar class
struct Bar
: private Foo
{
  /// public member function
  void baz() { }
};
````

And this config:
````
EXTRACT_PRIVATE = NO
INLINE_INHERITED_MEMB = YES
````

I get this output with Doxygen 1.18.4
````
<p>This is my bar class.
 <a href="structBar.html#details">More...</a></p>
<div class="dynheader">
Inheritance diagram for Bar:</div>
<div class="dyncontent">
 <div class="center">
  <img src="structBar.png" usemap="#Bar_map" alt=""/>
  <map id="Bar_map" name="Bar_map">
<area href="structFoo.html" title="This is my foo class. " alt="Foo" shape="rect" coords="0,0,34,24"/>
</map>
 </div></div>
<table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pub-methods"></a>
Public Member Functions</h2></td></tr>
<tr class="memitem:aed0079f81c54fe079eca5b25478a8e87"><td class="memItemLeft" align="right" valign="top"><a id="aed0079f81c54fe079eca5b25478a8e87"></a>
void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="structBar.html#aed0079f81c54fe079eca5b25478a8e87">baz</a> ()</td></tr>
<tr class="memdesc:aed0079f81c54fe079eca5b25478a8e87"><td class="mdescLeft">&#160;</td><td class="mdescRight">public member function <br /></td></tr>
<tr class="separator:aed0079f81c54fe079eca5b25478a8e87"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pri-types"></a>
Private Types</h2></td></tr>
<tr class="memitem:a0975ea1542b18a60760599de38453a50"><td class="memItemLeft" align="right" valign="top"><a id="a0975ea1542b18a60760599de38453a50"></a>
typedef int&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="structFoo.html#a0975ea1542b18a60760599de38453a50">value_type</a></td></tr>
<tr class="memdesc:a0975ea1542b18a60760599de38453a50"><td class="mdescLeft">&#160;</td><td class="mdescRight">public typedef <br /></td></tr>
<tr class="separator:a0975ea1542b18a60760599de38453a50"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pri-methods"></a>
Private Member Functions</h2></td></tr>
<tr class="memitem:adbe282a1cc9e0d276848775c9799fb23"><td class="memItemLeft" align="right" valign="top"><a id="adbe282a1cc9e0d276848775c9799fb23"></a>
void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="structFoo.html#adbe282a1cc9e0d276848775c9799fb23">m_impl</a> ()</td></tr>
<tr class="memdesc:adbe282a1cc9e0d276848775c9799fb23"><td class="mdescLeft">&#160;</td><td class="mdescRight">public function <br /></td></tr>
<tr class="separator:adbe282a1cc9e0d276848775c9799fb23"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table>
<a name="details" id="details"></a><h2 class="groupheader">Detailed Description</h2>
<div class="textblock"><p>This is my bar class. </p>
</div><hr/>The documentation for this struct was generated from the following file:<ul>
<li>t2.cc</li>
</ul>
````

Note that the inherited members are shown, despite `EXTRACT_PRIVATE=NO`. They have the correct access, i.e. they are private in the derived class despite being public in the base class, but they shouldn't be shown at all.

As a workaround I can use:

````
struct Bar
/** @cond */ : private Foo /** @endcond */
{
````

but this clutters up the source with visual noise that shouldn't be needed. The members of the private base class are not part of the public API I'm trying to document.

**Comment by albert-github** [comment]
You mention as version: 1.18.4, but this is probably a typo and should either be 1.8.4 or 1.8.14 ?

Can you please attach a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem?  Please don't add external links as they might not be persistent. (so there is no confusion over settings and used code).

**Comment by jwakely** [comment]
Yes, sorry - 1.8.14 is what I meant.

The source and config shown above are complete, and exactly what I'm testing with. Do you really need a tar file containing them?

**Comment by albert-github** [comment]
Normally that would be nice, so we can just download it and untar it. Now we have to cut and paste etc.

With your source I can reproduce the problem tough.

**Comment by jwakely** [comment]
[6968.tar.gz](https://github.com/doxygen/doxygen/files/3152673/6968.tar.gz)

As requested. Just run `doxygen` in the 6968 directory.

**Comment by jwakely** [comment]
Understood - I'll provide tar files in future. Thanks for looking at it so quickly.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #6970

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of a release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.16.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
